.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,7 +19,7 @@
 [ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
 [ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
 [ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
-[ ] Add autofixing to a rule
+[ ] Add auto fixing to a rule
```

Comment:
I would personally consider `autofix` to be a single word. In our codebase, it's also more common to refer to it as a single word.

Reply:
Makes sense to me 👍

Reply:
Yeah, we generally use “autofix”, so I’d say let’s keep this as-is.

Reply:
```suggestion
[ ] Add autofix to a rule
```